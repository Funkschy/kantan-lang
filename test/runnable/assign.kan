import "io";

def main() {
    let x = 5;

    x += 1;
    io.printf("%d\n", x);
    x *= 2;
    io.printf("%d\n", x);
    x /= 2;
    io.printf("%d\n", x);
    x -= 3;
    io.printf("%d\n", x);
    x %= 2;
    io.printf("%d\n", x);

    let y = x += 1;
    io.printf("%d %d\n", y, x);
}
