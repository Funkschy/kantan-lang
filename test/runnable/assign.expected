6
12
6
3
1
2 2
